src/joiners.R

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env Rscript
+
+library(lubridate)
+library(dplyr)
+library(MASS)
+library(fitdistrplus)
+library(FAdist)
+library(glm2)
+
+args = commandArgs(trailingOnly=TRUE)
+input <- args[1]
+model.out <- args[2]
+rates.out <- args[3]
+seed.long <- args[4]
+set.seed(seed.long)
+
+# input <- '/var/folders/tw/tzkjfqd11md5hdjcyjxgmf740000gn/T/file726180705789120480.csv'
+# model.out <- '/var/folders/tw/tzkjfqd11md5hdjcyjxgmf740000gn/T/file8865703702117111587.csv'
+# rates.out <- '/var/folders/tw/tzkjfqd11md5hdjcyjxgmf740000gn/T/file2591792755455537745.csv'
+
+df <- read.csv(input, header = TRUE, stringsAsFactors = FALSE, na.strings ='')
+df$beginning <- as.Date(parse_date_time(df$beginning, 'ymd HMS'))
+df$admission_age <- as.factor(as.character(df$admission_age))
+
+diffs <- df %>% arrange(admission_age, beginning) %>% group_by(admission_age) %>%
+  mutate(diff = interval(lag(beginning), beginning) / days(1), n = n()) %>%
+  ungroup %>% filter(!is.na(diff) & n >= 3) %>% # We need at least 3 data points for each age to generate 2 diffs
+  dplyr::select(admission_age, diff, beginning) %>% mutate(diff = diff + 0.01) %>%
+  mutate(admission_age = droplevels(admission_age)) %>%
+  as.data.frame
```

Comment:
sorry for being pedantic, could you follow each pipe (`%>%`) with a newline for ease of reading please